charts/invenio/templates/sentry-secret.yaml

```diff
@@ -1,15 +1,16 @@
-{{- if and (.Values.invenio.sentry.enabled) (not .Values.invenio.sentry.existing_secret) }}
+{{- if and (.Values.invenio.sentry.enabled) (and (not .Values.invenio.sentry.existingSecret) (not .Values.invenio.sentry.existing_secret))}}
+{{- $dsn := .Values.invenio.sentry.dsn | required ".Values.invenio.sentry.dns is required if not secret is provided." -}}
```

Comment:
```suggestion
{{- $dsn := .Values.invenio.sentry.dsn | required ".Values.invenio.sentry.dns is required unless secret provided." -}}
```

Reply:
I wrote ".Values.invenio.sentry.dns is required unless existingSecret is provided." I find it more explicit. WDYT?

Reply:
Much better! I mostly corrected you because "if not secret is" sounded incorrect.  Your new version is great.